### PR TITLE
alert-notification box and loading bar

### PR DESCRIPTION
**Pull request by rawjazz**
a user can access all alerts in a notification box on the dashboard 
each site corresponding with alerts not silenced on the site
and on click of a row he is redirected to the alerts panel of a
corresponding site

### CODE REVIEW

**Comment by jlark** [comment]
Hi Roger,
Looks like you have conflicts, please merge these

Step 1: From your project repository, bring in the changes and test.
`
git fetch origin
git checkout -b alert-bagde origin/alert-bagde
git merge master`
Step 2: Merge the changes and update on GitHub.

`git checkout master
git merge --no-ff alert-bagde
git push origin master`

File: seshdash/sesh-ui/js/sesh_dashboard.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ $('#containerhigh').highcharts({
                 }
             },
             labels: {
-                format: '{value} Wh',
+                format: '{value} HW',
```
Comment (jlark) [inline seshdash/sesh-ui/js/sesh_dashboard.js]: can you modify this to be Wh? I think this was changed any you merged it back somehow.

File: seshdash/sesh-ui/js/sesh_dashboard.js [hunk 2]
```diff
@@ -174,6 +164,38 @@ get_high_chart( date, HighChartHighPvProduction, HighChartHighCloudCover);
 
   setTimeout(setModalLoad, initiatingModalTime);
 
+
+
+
+    $(document).ready(function(){
+    alertId = $(this).attr('classid');
+    console.log("alertId is " + alertId)
```
Comment (jlark) [inline seshdash/sesh-ui/js/sesh_dashboard.js]: can you remove console logging?

File: seshdash/static/sesh-ui/js/sesh_dashboard.js [hunk 3]
```diff
@@ -32,7 +32,22 @@ Morris.Bar({
   labels: ['Cloud Cover %']
 });
 */
+//nano bar
+        var nanoBar=function(){
+        var options={
+              bg :"#4AA0BD",
```
Comment (jlark) [inline seshdash/static/sesh-ui/js/sesh_dashboard.js]: is it possible to put this color in the CSS?

File: seshdash/views.py [hunk 4]
```diff
@@ -575,6 +574,28 @@ def get_alerts(request):
 
     return HttpResponse(json.dumps(alert_data))
 
+    
+
+
```
Comment (jlark) [inline seshdash/views.py]: this requires the @login_required decorator as anyone that has logged in can only view alerts

File: seshdash/views.py [hunk 5]
```diff
@@ -575,6 +574,28 @@ def get_alerts(request):
...
+def get_notifications_alerts(request):
+    
+    sites = Sesh_Site.objects.all()
```
Comment (jlark) [inline seshdash/views.py]: We can't use `Sesh_Site.objects.all()` you need to fetch the sites that the user has permission to see.

**Comment by rawjazz** [comment]
tests are fixed

File: seshdash/urls.py [hunk 6]
```diff
@@ -16,9 +16,9 @@
             url(r'^api-token-auth/', authtokenviews.obtain_auth_token),
             url(r'^users/$', views.UserList.as_view()),
             url(r'^users/(?P<pk>[0-9]+)/$', views.UserDetail.as_view()),
-            url(r'^get-alert-data', views.display_alert_data),
```
Comment (jlark) [inline seshdash/urls.py]: I think this needed

File: seshdash/views.py [hunk 7]
```diff
@@ -575,6 +575,28 @@ def get_alerts(request):
...
 
+    
+
+@login_required
+def get_notifications_alerts(request):
+    
+    sites =_get_user_sites(request)
+
+    arr = []
+
```
Comment (jlark) [inline seshdash/views.py]: In the future no need for so much white space

**Comment by jlark** [comment]
Ok, looks good thanks @rawjazz

**Pull request merged**